chsdi/lib/filters.py

```diff
@@ -11,6 +11,7 @@ def full_text_search(query, ormColumns, searchText):
     def ilike_search(col):
         if col is not None:
             return cast(col, Text).ilike('%%%s%%' % searchText)
+    # TODO: clean-up when only Python 3.x and no longer 2.x is in use
```

Comment:
Probably this comment doesn't make sense here ? If I'm not missing something the code is compatible for both python2 and python3 and there is nothing to cleanup.